### PR TITLE
add marble and atomic assets to tools list - develop

### PR DESCRIPTION
**Pull request by duncand0nuts**

### CODE REVIEW

File: docs/05_community-developer-tools/index.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ If you would like to submit a tool to be added to the list, please fill in [this
 
 * [dGoods](https://github.com/MythicalGames/dgoods): An open source and free standard for handling the virtual representation of items, both digital and physical.
 * [SimpleAssets](https://github.com/CryptoLions/SimpleAssets): A simple standard for digital assets on EOSIO blockchains: Non-Fungible Tokens (NFTs), Fungible Tokens (FTs), and Non-Transferable Tokens (NTTs).
+* [Marble](https://github.com/Dappetizer/marble-digital-items): Marble is a lightweight, highly customizable digital item standard for just about anything.
```
Comment (iamveritas) [inline docs/05_community-developer-tools/index.md:31]: for just about anything?... this sounds cheesy, cheeky; can we reformulate it to be more specific?

**Review by iamveritas — CHANGES_REQUESTED** [review]
I made one comment. Everything else looks great.

File: docs/05_community-developer-tools/index.md (hunk 1 above)
Comment (halsaphi) [reply docs/05_community-developer-tools/index.md:31]: I agree with Ovi, seems a little bit vague, can it make my morning tea?

Comment (halsaphi) [reply docs/05_community-developer-tools/index.md:31]: maybe "most" should be "many"

**Comment by duncand0nuts** [comment]
@iamveritas I have removed the end of the description which you commented on. https://github.com/EOSIO/welcome/pull/299/commits/2d6e74c87476094a958e0e9397945bbbeaf041f8

**Review by halsaphi — APPROVED** [review]

**Review by iamveritas — APPROVED** [review]

**Review by lparisc — APPROVED** [review]
This PR should also be applied to `release/2.0.x` explicitly, since now `develop` targets next versions only (2.1 in this case). @duncand0nuts

**Pull request merged**